src/main/java/frc/robot/Drive.java

```diff
@@ -134,32 +135,32 @@ public static void initializeDrive(){
     }
    
    public static void arcadeDrive(double side, double forward, boolean isShifted){
-        R1.set(side - forward);
-		L1.set(side + forward);
-		Shiftersint(isShifted);
-	}
-	
-	public static void Shiftersint (boolean isShifted){
-		Shifters = new DoubleSolenoid(ShiftersForward, ShiftersReverse);
-		if (isShifted == true){
-			Shifters.set(Value.kReverse);
-		}else {
-			Shifters.set(Value.kForward);
-		}
-	}
-	public static void setShifters(){
-		if (driveStick.getRawButton(9)){
-			Shiftersint(true);
-		}else{
-			Shiftersint(false);
-		}
+        R1.set(side + forward);
+		L1.set(side - forward);
+		Robot.Shifting(isShifted);
 	}
 	
 	// if pressure starts to get low, it will activate the compressor
 	public static void Compressor() {
 		compressor.setClosedLoopControl(compressor.getPressureSwitchValue());
 	}
+
+	public static void Climber(){
+		C1 = new TalonSRX(11);
+		C2 = new TalonSRX(12);
+		C3 = new TalonSRX(13);
+		if (driveStick.getRawButton(4)){
+			C1.set(ControlMode.PercentOutput, .5);
+			C2.set(ControlMode.PercentOutput, .5);
+			C3.set(ControlMode.PercentOutput, .5);
```

Comment:
0.5 magic number. Should be extracted to a const/final in the class so it is easier to modify.